manifests/0000_70_cluster-network-operator_02_rbac.yaml

```diff
@@ -1,16 +1,48 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cluster-network-operator-service-account
+  namespace: openshift-network-operator
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/single-node-developer: "true"
+
+---
+
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: default-account-cluster-network-operator
```

Comment:
Since you no longer need this, you need to add a tombstone so that CVO cleans it up for clusters that are upgraded, https://github.com/openshift/enhancements/blob/master/dev-guide/cluster-version-operator/dev/object-deletion.md

Reply:
IIUC this is already handled here: https://github.com/openshift/cluster-network-operator/pull/1808/files/667ed4572550658c110c0070b77d52962aa290d0#diff-2edaaa8110ef6986976b6cb27708734a5ed180a2da91b849e8f5f59528cc42b6R37